### PR TITLE
Add tinyicon plugin to allow favicon to be badged

### LINKED ISSUE #279: Add ability for favicon to be badged
Options:

http://lab.ejci.net/favico.js/
https://github.com/tommoor/tinycon

### PR DESCRIPTION
**Pull request by Stanton**
- [x] Safari
- [x] Chrome
- [x] Firefox
- [x] IE8
- [x] IE9
- [x] IE10
- [x] IE11

Fixes #279 

**Chrome, Firefox, Opera**
screen shot 2016-08-28 at 11 23 40

**IE, Safari**
screen shot 2016-08-28 at 11 25 02

### CODE REVIEW

**Comment by Stanton** [comment]
Added docs here: https://jadu.gitbooks.io/pulsar/content/favicons.html

File: views/pulsar/components/favicons-cms.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <link rel="mask-icon" href="/jadu/bundles/pulsar/images/favicons/cms/safari-pinned-tab.svg" color="#15a6d1">
+<link rel="mask-icon" href="/jadu/bundles/pulsar/images/favicons/cms/safari-pinned-tab.svg" color="#15a6d1">
```
Comment (langy) [inline views/pulsar/components/favicons-cms.html]: Duplicates line above

File: views/pulsar/components/favicons-cp.html [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <link rel="mask-icon" href="/images/favicons/cp/safari-pinned-tab.svg" color="#_d_c5172">
+<link rel="mask-icon" href="/images/favicons/cp/safari-pinned-tab.svg" color="#_d_c5172">
```
Comment (langy) [inline views/pulsar/components/favicons-cp.html]: Duplicates line above

File: views/pulsar/components/favicons-cxm.html [hunk 3]
```diff
@@ -1,4 +1,5 @@
 <link rel="mask-icon" href="/images/favicons/cxm/safari-pinned-tab.svg" color="#54_b9_a9">
+<link rel="mask-icon" href="/images/favicons/cxm/safari-pinned-tab.svg" color="#54_b9_a9">
```
Comment (langy) [inline views/pulsar/components/favicons-cxm.html]: Duplicates line above

File: views/pulsar/components/favicons-xfp.html [hunk 4]
```diff
@@ -1,4 +1,5 @@
 <link rel="mask-icon" href="/images/favicons/xfp/safari-pinned-tab.svg" color="#80_b_a27">
+<link rel="mask-icon" href="/images/favicons/xfp/safari-pinned-tab.svg" color="#80_b_a27">
```
Comment (langy) [inline views/pulsar/components/favicons-xfp.html]: You get the idea

**Comment by langy** [comment]
Apart from comments, 👍

**Comment by Stanton** [comment]
They're autogenerated files so should just need clearing and the task running again

**Pull request merged**